yktrace/src/sir.rs

```diff
@@ -105,23 +105,26 @@ lazy_static! {
 #[derive(Debug, Hash, Eq, PartialEq)]
 pub struct SirLoc {
     pub symbol_name: String,
-    pub bb_idx: u32
+    pub bb_idx: u32,
+    pub addr: u64,
 }
 
 impl From<&CoreSirLoc> for SirLoc {
     fn from(core_loc: &CoreSirLoc) -> SirLoc {
         SirLoc {
             symbol_name: String::from_utf8(core_loc.symbol_name().to_vec()).unwrap(),
-            bb_idx: core_loc.bb_idx()
+            bb_idx: core_loc.bb_idx(),
+            addr: 0 // Does CoreSirLoc need addr too?
```

Comment:
Any idea about this `CoreSirLoc` question here?

Reply:
No, no idea. What is `CoreSirLoc` used for?

Reply:
I had to go digging to remind myself.

It's a mirror of `SirLoc` used only in software tracing. With SWT `libcore` had to fabricate SIR locations at runtime, and you can't use `String` in `core`, so it's a `SirLoc` with a fixed size array instead of a string.

Since we are not looking at software tracing, I'd just remove the comment.

Reply:
This is still outstanding I think.

Reply:
Whoops, I changed the `0` to `None` but forgot to remove the comment. Fixed in latest commit.